test/views/arrival/P5/ArrivalNotificationWithFunctionalErrorsP5ViewSpec.scala

```diff
@@ -78,23 +74,15 @@ class ArrivalNotificationWithFunctionalErrorsP5ViewSpec
 
   behave like pageWithPagination(controllers.testOnly.routes.ArrivalNotificationWithFunctionalErrorsP5Controller.onPageLoad(None, arrivalIdP5).url)
 
-  behave like pageWithSummaryLists()
+  behave like summaryLists
```

Comment:
Can use the new view behaviour for this too

Reply:
done